tsconfig.json

```diff
@@ -20,6 +20,6 @@
       "@components/*": ["components/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "lib/tests/s3-upload.test.js"],
```

Comment:
This was probably an auto addition by typescript module of VSCode. 'js' files don't need to be transcompiled by the Typescript engine.